src/obclient/CMakeLists.txt

```diff
@@ -1,15 +1,25 @@
-PROJECT(obclient)
-MESSAGE("Begin to build " ${PROJECT_NAME})
-MESSAGE(STATUS "This is PROJECT_BINARY_DIR dir " ${PROJECT_BINARY_DIR})
-MESSAGE(STATUS "This is PROJECT_SOURCE_DIR dir " ${PROJECT_SOURCE_DIR})
+ADD_EXECUTABLE(obclient)
+MESSAGE("Begin to build " obclient)
 
+INCLUDE(CheckIncludeFiles)
 
 #INCLUDE_DIRECTORIES([AFTER|BEFORE] [SYSTEM] dir1 dir2 ...)
-INCLUDE_DIRECTORIES(. ${PROJECT_SOURCE_DIR}/../../deps /usr/local/include SYSTEM)
+TARGET_INCLUDE_DIRECTORIES(obclient PRIVATE . ${PROJECT_SOURCE_DIR}/../deps /usr/local/include /usr/include)
```

Comment:
1. why not use ${PROJECT_SOURCE_DIR} replacing "object PRIVATE"?
2. why not use SYSTEM replacing "/usr/include"